app/src/App.js

```diff
@@ -58,6 +61,53 @@ function App() {
   //   setEditMode(!editMode)
   // }
 
+  const SideContent = () => (
```

Comment:
I've seen lots of problems caused by defining components within other components.

Every time React renders `App`, it's going to redefine `SideContent`, and that means that all elements rendered as part of SideContent will be removed from the DOM and re-added. This is incredibly inefficient and can cause big UX problems – in Projects, we had a spot where an input was removed from the DOM and re-added after every keystroke! For the user, this meant that the input lost focus every time they typed a character in it! Incredibly frustrating.

I get that it's tempting to do this. We don't have to pass in any props! But the tradeoffs are almost never worth it.